google-feeds.html

```diff
@@ -96,7 +96,7 @@
     queryChanged: function() {
       if (this.query) {
         this.loading = true;
-        this.findFeeds();
+        this.withFeedsApi(this.findFeeds.bind(this));
```

Comment:
you could set this as a private prop in `ready()`. That way it's not bound every query change.

`this._feedHandler = this.findFeeds.bind(this);`